packages/fxa-content-server/server/lib/routes/get-index.js

```diff
@@ -55,6 +55,9 @@ module.exports = function (config) {
   const FEATURE_FLAGS_RESET_PWD_WITH_CODE = config.get(
     'featureFlags.resetPasswordWithCode'
   );
+  const FEATURE_FLAGS_RECOVERY_CODE_SETUP_ON_SYNC_SIGN_IN = config.get(
```

Comment:
Missed this when adding feature flag orignally. This serves the feature flag on the backbone version of the app, which is what we use initially to get config values.